jerry-core/ecma/builtin-objects/ecma-builtin-global.cpp

```diff
@@ -112,7 +112,7 @@ ecma_builtin_global_object_parse_int (ecma_value_t this_arg __attr_unused___, /*
   ecma_string_t *number_str_p = ecma_get_string_from_value (string_var);
   lit_utf8_size_t str_size = ecma_string_get_size (number_str_p);
 
-  MEM_DEFINE_LOCAL_ARRAY (utf8_string_buff, str_size, lit_utf8_byte_t);
+  MEM_DEFINE_LOCAL_ARRAY (utf8_string_buff, str_size + 1, lit_utf8_byte_t);
```

Comment:
@ruben-ayrapetyan, yeah that's why I was thinking initially, thus I have this: https://github.com/galpeter/jerryscript/commit/f3081a005fd91be09b151f725a23185489c05b3c  would that be ok?